pandas/core/strings.py

```diff
@@ -734,6 +734,33 @@ def str_split(arr, pat=None, n=None):
     return res
 
 
+def str_rsplit(arr, pat=None, n=None):
+    """
+    Split each string in the Series/Index by the given delimiter
+    string, starting at the end of the string and working to the front.
+    Equivalent to :meth:`str.rsplit`.
+
```

Comment:
We're starting to use ``versionadded`` directive (#10215). Please add ``.. versionadded:: 0.16.2``, like:

- https://github.com/pydata/pandas/blob/master/pandas/core/generic.py#L2051